codegen/masm2/src/artifact.rs

```diff
@@ -94,3 +100,248 @@ impl Rodata {
         Ok(felts)
     }
 }
+
+inventory::submit! {
+    midenc_session::CompileFlag::new("test_harness")
```

Comment:
I suspect this causes an error `-> Command midenc: Argument names must be unique, but 'test_harness' is in use by more than one argument or group` when I run tests locally. Interestingly, it does not happen on CI (cached old version of `clap_builder`?).

Reply:
Ah, that's happening because the test crate is linking both `midenc-hir` and `midenc-hir2` which both define the same flag.